### PR TITLE
filter showInNavigationMenu on activeComponents and show 1 element directly if it is the only one

### PR DESCRIPTION
**Pull request by martinapippi**
Closes https://issuetracker.deltares.nl/browse/DWO-657?filter=-1 and https://issuetracker.deltares.nl/browse/DWO-682?filter=-1

### CODE REVIEW

File: src/layouts/DefaultLayout.vue [hunk 1]
```diff
@@ -50,6 +50,19 @@
             </v-list-item>
           </v-list>
         </v-menu>
+        <v-list density="compact" v-if="navigationMenuComponents.length === 1">
+          <v-list-item
+            :value="navigationMenuComponents[0]"
+            :to="navigationMenuComponents[0].to"
```
Comment (hvangeffen) [inline src/layouts/DefaultLayout.vue:56]: This errors as this is now immediately created on page load for a dynamic route that has not been setup yet: 
image

As such we need to check if the route exists:
```vue
<v-list-item
    v-if="router.hasRoute(navigationMenuComponents[0].to.name)"
```

File: src/views/AboutView.vue [hunk 2]
```diff
@@ -3,7 +3,8 @@
     <v-card-title class="justify-center">
       {{ configStore.general.title ?? 'Delft-FEWS Web OC' }}
     </v-card-title>
-    <v-card-text v-if="configStore.activeComponents.length > 0">
+    <v-card-text v-if="configStore.activeComponents.length > 0"> </v-card-text>
+    <v-card-text v-else-if="configStore.activeComponents.length > 1">
       Select one of the following options to get started.
     </v-card-text>
     <v-card-text v-else>
```
Comment (hvangeffen) [inline src/views/AboutView.vue:10]: Empty v-card-text seems weird here, could make it:
```suggestion
    <v-card-text v-else-if="configStore.activeComponents.length > 1">
      Select one of the following options to get started.
    </v-card-text>
    <v-card-text v-else-if="configStore.activeComponents.length === 0">
```

File: src/views/AboutView.vue [hunk 3]
```diff
@@ -12,6 +13,7 @@
     </v-card-text>
     <v-list density="compact">
       <v-list-item
+        v-if="configStore.activeComponents.length > 1"
```
Comment (hvangeffen) [inline src/views/AboutView.vue:16]: This if should probably go on the `v-list`

File: src/layouts/DefaultLayout.vue [hunk 4]
```diff
@@ -50,6 +50,19 @@
             </v-list-item>
           </v-list>
         </v-menu>
+        <v-list density="compact" v-if="navigationMenuComponents.length === 1">
```
Comment (hvangeffen) [inline src/layouts/DefaultLayout.vue:53]: Should this still be a `v-list` and should it look different than the style used for the menu button above?:
```vue
            <v-list-item
              aria-label="Menu button"
              v-bind="props"
              class="ma-1"
              rounded
              variant="tonal"
            >
```

**Comment by hvangeffen** [comment]
Also the pptx mentions:
image

Now the v-list only shows on single component, shouldn't there be a `HtmlDisplay` check?

File: src/views/AboutView.vue (hunk 2 above)
Comment (martinapippi) [reply src/views/AboutView.vue:10]: if the configStore.activeComponents.length is <= 1 then nothing should be shown, so also if it is === 1

File: src/layouts/DefaultLayout.vue (hunk 4 above)
Comment (martinapippi) [reply src/layouts/DefaultLayout.vue:53]: I assumed so, yes

**Review by hvangeffen — APPROVED** [review]

**Pull request merged**